src/main/java/com/minecolonies/coremod/client/gui/WindowTownHall.java

```diff
@@ -320,7 +463,233 @@ private void updateWorkOrders()
     {
         workOrders.clear();
         workOrders.addAll(townHall.getColony().getWorkOrders());
-        Collections.sort(workOrders, (first, second) -> second.getPriority() > first.getPriority() ? 1 : (second.getPriority() < first.getPriority() ? -1 : 0));
+        workOrders.sort((first, second) -> second.getPriority() > first.getPriority() ? 1 : (second.getPriority() < first.getPriority() ? -1 : 0));
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Extract this nested ternary operation into an independent statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS3358)